src/models/__tests__/helpers/mockCloudagent.ts

```diff
@@ -0,0 +1,28 @@
+import { container } from 'tsyringe'
+import { Dispatcher, MockAgent, getGlobalDispatcher, setGlobalDispatcher } from 'undici'
+
+import { Env } from '../../../env.js'
+import { successResponse } from '../fixtures/cloudagentFixtures.js'
+
+const env = container.resolve(Env)
+
+export function withCloudagentMock(code: number = 200, responseBody: any = successResponse) {
+  let originalDispatcher: Dispatcher
+  let agent: MockAgent
+  beforeEach(function () {
+    originalDispatcher = getGlobalDispatcher()
+    agent = new MockAgent()
+    setGlobalDispatcher(agent)
+
+    const client = agent.get(env.get('CLOUDAGENT_ADMIN_ORIGIN'))
+    client
+      .intercept({
+        path: `/oob/create-invitation`,
```

Comment:
yay! first interaction (even if it's a mock)

Reply:
works for reals as well!